problems/1.types+operators.js

```diff
@@ -2,16 +2,23 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (typeof(val) === "number"){
+    return true
```

Comment:
Your logic is correct, but you can simple just return typeof val (typeof doesn't require parentheses) === "number" and you can get your answer. This will return true or false depending on the outcome of the conditional